### PR TITLE
Send CB bootstrap API calls thru docker exec

### PR DESCRIPTION
**Pull request by tgross**
@misterbisson this PR addresses https://github.com/tgross/triton-touchbase/issues/3 by passing the `removeIndex` and `createIndex` functions thru `docker exec`.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -117,9 +119,10 @@ createBucket() {
 # send a REST API call to Couchbase to create a N1QL index
 createIndex() {
     echo $1
-    curl -s --fail -s -X POST http://${N1QLAPI}/query/service \
-         -u ${CB_USER}:${CB_PASSWORD} \
-         -d "statement=$1"
+    docker exec -it ${PREFIX}_couchbase_1 \
+           curl -s --fail -s -X POST http://${N1QLAPI}/query/service \
```
Comment (misterbisson) [inline start.sh]: Doubled `-s`

**Comment by misterbisson** [comment]
Small question about a doubled arg, and a shaken :facepunch: at Couchbase for the inconsistent support between CLI tools and API, otherwise lgtm.

File: start.sh (hunk 1 above)
Comment (tgross) [inline start.sh]: Fixed.

**Comment by tgross** [comment]
Fixed that extra `-s`. Going to merge it.

**Pull request merged**